official/utils/testing/scripts/presubmit.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Presubmit script that run tests and lint under local environment.
+# Make sure that tensorflow and pylint is installed.
+# usage: models >: ./official/utils/testing/scripts/presubmit.sh
+set +x
+
+# Assume the pwd is under models.
+cd official
+
+exit_code=0
+
+# Check lint:
+RC_FILE="utils/testing/pylint.rcfile"
+
+echo "===========Running lint test============"
+for file in `find . -name '*.py' ! -name '*test.py' -print`
+do
+  echo "Linting ${file}"
+  pylint --rcfile="${RC_FILE}" "${file}" || exit_code=$?
```

Comment:
If you change the `cd` at the top, you should also inject the absolute path into the rcfile path. It has trouble finding it otherwise. (Not sure if there is some weird `cd` related bash behavior.)

Reply:
Ack.